myvpn-client/main.go

```diff
@@ -90,11 +122,19 @@ func main() {
 			}
 			glog.V(3).Infoln("recieve IP packet from tun", b[:n])
 
-			_, err = conn.Write(b[:n])
-			if err != nil {
-				glog.Errorln("fail to write to connection", err)
-				errc <- err
+			for {
+				_, err = conn.Write(b[:n])
```

Comment:
This commit is a quick fix. I assume when `conn.Write` got an error `conn.Read` will also got an error. Then it will close the connection and trying to dial a new connection.

https://github.com/cirias/myvpn/pull/5/files#diff-ffe3cc8e1190a5af08d2c0aa4ff777c5R73